### ISSUE TITLE
Backport fix for IPSCCP crash

### ISSUE THREAD

**Issue by nikic** [llvm/llvm-project#55304]
/cherry-pick 857c612d899f271402470d4026d2e3be1dce53a4

cc @fhahn

**Comment by fhahn** [comment]
Should be straight-forward to pick for the release branch.

**Comment by nikic** [comment]
The cherry-pick workflow failed with a TypeError (https://github.com/llvm/llvm-project/runs/6320780928?check_suite_focus=true)
```

cherry-picking ['857c612d899f271402470d4026d2e3be1dce53a4']
Traceback (most recent call last):
  File "./llvm/utils/git/github-automation.py", line 294, in <module>
    if not release_workflow.execute_command():
  File "./llvm/utils/git/github-automation.py", line 247, in execute_command
    return self.create_branch(args.split())
  File "./llvm/utils/git/github-automation.py", line 177, in create_branch
    local_repo.git.checkout(self.release_branch_for_issue)
  File "./llvm/utils/git/github-automation.py", line 123, in release_branch_for_issue
    m = re.search('branch: (.+)',milestone.description)
  File "/usr/lib/python3.8/re.py", line 201, in search
    return _compile(pattern, flags).search(string)
TypeError: expected string or bytes-like object
```

**Comment by tstellar** [comment]
Merged: 60c8e02c9d12
